include/dt-bindings/dt-binding.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_DT_BINDING_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_DT_BINDING_H_
+
```

Comment:
Add a comment about why we have this include file just including <sys/util_macro.h>